x/evm/keeper/state_transition.go

```diff
@@ -208,7 +203,9 @@ func (k *Keeper) ApplyTransaction(tx *ethtypes.Transaction) (*types.MsgEthereumT
 // The preprocessing steps performed by the AnteHandler are:
 //
 // 1. set up the initial access list (iff fork > Berlin)
-func (k *Keeper) ApplyMessage(evm *vm.EVM, msg core.Message, cfg *params.ChainConfig) (*types.MsgEthereumTxResponse, error) {
+
+// NOTE: eth_call/eth_estimateGas don't do ante handler, so shouldn't do gas refund either.
```

Comment:
remove the new line, otherwise, the comment above won't be displayed. Also, update it to add more context about the `simulate` field that has been added